From d225aaa8841f47ba8aa7b353b0ac3028d5913efe Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 9 Feb 2017 01:27:49 -0800
Subject: [PATCH] caps-abbrev.awk: fix gawk's path

It should be /usr/bin/gawk as other scripts use in this package.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>

---
 tests-m32/caps-abbrev.awk  | 2 +-
 tests-mx32/caps-abbrev.awk | 2 +-
 tests/caps-abbrev.awk      | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests-m32/caps-abbrev.awk b/tests-m32/caps-abbrev.awk
index c00023b..a56cd56 100644
--- a/tests-m32/caps-abbrev.awk
+++ b/tests-m32/caps-abbrev.awk
@@ -1,4 +1,4 @@
-#!/bin/gawk
+#!/usr/bin/gawk
 #
 # This file is part of caps strace test.
 #
diff --git a/tests-mx32/caps-abbrev.awk b/tests-mx32/caps-abbrev.awk
index c00023b..a56cd56 100644
--- a/tests-mx32/caps-abbrev.awk
+++ b/tests-mx32/caps-abbrev.awk
@@ -1,4 +1,4 @@
-#!/bin/gawk
+#!/usr/bin/gawk
 #
 # This file is part of caps strace test.
 #
diff --git a/tests/caps-abbrev.awk b/tests/caps-abbrev.awk
index c00023b..a56cd56 100644
--- a/tests/caps-abbrev.awk
+++ b/tests/caps-abbrev.awk
@@ -1,4 +1,4 @@
-#!/bin/gawk
+#!/usr/bin/gawk
 #
 # This file is part of caps strace test.
 #
